### PR TITLE
:bug: fix bad quote in link

### LINKED ISSUE #13938: Bad link in amp-img.md making gulp check-links fail
`https://www.ampproject.org/docs/guides/responsive/control_layout#what-if-the-layout-attribute-isn’t-specified?`

should be

`https://www.ampproject.org/docs/design/responsive/control_layout#what-if-the-layout-attribute-isn%E2%80%99t-specified?`

### PR DESCRIPTION
**Pull request by jpettitt**
:bug: Fixes #13938

### CODE REVIEW

File: builtins/amp-img.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ If you want your image to display at a fixed size:
 2. Specify the `width` and `height`.
 
 {% call callout('Read on', type='read') %}
-Learn about the [inferred layout](https://www.ampproject.org/docs/guides/responsive/control_layout#what-if-the-layout-attribute-isn’t-specified?) if you don't specify the `layout` attribute.
+Learn about the [inferred layout](https://www.ampproject.org/docs/guides/responsive/control_layout#what-if-the-layout-attribute-isn%E2%80%99t-specified?) if you don't specify the `layout` attribute.
```
Comment (rsimha) [inline builtins/amp-img.md]: I don't think it's necessary to use escaped text. From what I saw, this works with just a non-pretty apostrophe.

```
https://www.ampproject.org/docs/guides/responsive/control_layout#what-if-the-layout-attribute-isn't-specified?
```

Comment (jpettitt) [reply builtins/amp-img.md]: No it doesn't - if you use the simple apostrophe you get top of page not the anchor tag.

Comment (rsimha) [reply builtins/amp-img.md]: Aha, in that case, I stand corrected. (I merely checked to make sure the link checker was satisfied.)

Comment (jpettitt) [reply builtins/amp-img.md]: %$#% unicode :-)

Comment (jpettitt) [reply builtins/amp-img.md]: I don't have write, so if could merge it that would be great.

Comment (rsimha) [reply builtins/amp-img.md]: @jpettitt Looks like @bpaduch fixed this (among other things) in #13929, so this PR is now redundant.

Comment (jpettitt) [reply builtins/amp-img.md]: ok thx

**Pull request closed without merging**